benchmark_np.py

```diff
@@ -26,22 +24,29 @@ class AudioFolder(object):
     def __init__(
         self,
         root,
-        download=True,
-        extension='wav',
-        lib="librosa",
+        extension='wav'
     ):
         self.root = os.path.expanduser(root)
         self.data = []
         self.audio_files = get_files(dir=self.root, extension=extension)
-        self.loader_function = getattr(loaders, lib)
 
     def __getitem__(self, index):
-        return self.loader_function(self.audio_files[index])
+        return self.audio_files[index]
 
     def __len__(self):
         return len(self.audio_files)
 
 
+def test_np_loading(fp, lib):
+    import loaders
+    load_function = getattr(loaders, 'load_' + lib)
+    audio = load_function(fp)
+    if np.max(audio) > 0:
+        return True
+    else:
+        return False
```

Comment:
@bmcfee do you know a way to ignore measurements when using timeit?

Reply:
Hrm, never thought about it; not seeing an obvious workaround in the timeit API.

Reply:
￼OK so I guess the only alternative then it is to do a dry run in the beginning to just check if functions return of valid output for a given test file...